FRONTEND/fastparking/utils/get_news.py

```diff
@@ -1,6 +1,7 @@
 from pathlib import Path
 
 STORE_FILE = Path(__file__).resolve().parent.joinpath("news.txt")
+TELEGRAM_CHANEL_FILE = 'telegram_channel.txt'
```

Comment:
from utils.telegram_api import send_message_news

send_message_news(text)


PS set before .env TELEGRAM_TOKEN from examples and info on slack